packages/react-native/ReactAndroid/src/test/java/com/facebook/react/packagerconnection/JSPackagerClientTest.kt

```diff
@@ -0,0 +1,137 @@
+package com.facebook.react.packagerconnection
+
+import com.facebook.react.packagerconnection.ReconnectingWebSocket.ConnectionCallback
+import java.io.IOException
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class JSPackagerClientTest {
+  private fun createRH(action: String, handler: RequestHandler): Map<String, RequestHandler> {
```

Comment:
Since we are at this, let's call this `createRequestHandler`?

Plus let's move the `private` methods at the bottom after all the tests

Reply:
Since I'm going to change some semantics here, I noticed that the function is always called passing in the same `settings` and `clientId` as `test_client`. Wdyt encoding this as default arguments to the function, passing in on the call site only the handler?

Reply:
Yup that's also a valid point for improvement 👍

Reply:
I just pushed the changes, let me know if I have to revert something.